app/controllers/users/webauthn_setup_controller.rb

```diff
@@ -21,6 +21,10 @@ def new
       )
       result = form.submit(new_params)
       @platform_authenticator = form.platform_authenticator?
+      if @platform_authenticator && in_account_creation_flow? &&
+         current_user.webauthn_configurations.platform_authenticators.present?
```

Comment:
I wonder if this can be cleaned up for readability. What do you think about

```suggestion
if ft_unlock_setup_successful_on_account_setup
```

and then create the function as 
```suggestion
def ft_unlock_setup_successful_on_account_setup
    if @platform_authenticator && in_account_creation_flow? &&
         current_user.webauthn_configurations.platform_authenticators.present?
end
```
As long as the name denotes that the user has already set up F/T unlock on account creation.

Reply:
That's much cleaner, thanks!